src/parser/transform/constraint/transform_constraint.cpp

```diff
@@ -71,6 +74,9 @@ unique_ptr<Constraint> Transformer::TransformConstraint(duckdb_libpgquery::PGLis
 	case duckdb_libpgquery::PG_CONSTR_NULL:
 		return nullptr;
 	case duckdb_libpgquery::PG_CONSTR_DEFAULT:
+		if (column.type.IsSerial()) {
```

Comment:
Wasn't totally sure what to do in this case- arguably I could just silently let the default argument from the table declaration override the `nextval` value for the sequence that was created, but that seemed very odd.

Reply:
Same here - I would probably copy Postgres behavior